drivers/serial/uart_nrfx_uarte.c

```diff
@@ -80,6 +89,7 @@ struct uarte_async_cb {
 		gppi_channel_t ppi;
 		uint32_t cnt;
 	} rx_cnt;
+	int tx_amount;
```

Comment:
It should be `volatile` because it is read in the loop in `uarte_nrfx_poll_out()`.

Reply:
done